game.py

```diff
@@ -20,7 +20,31 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    wrong_guesses_count = 0
+    wrong_guesses_list = []
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    
+    while wrong_guesses_count < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+
+        if user_input in snowman_word and not correct_letter_guess_statuses[user_input]:
```

Comment:
Note that the `if user_guess in correct_letter_guess_statuses` is shorthand ([in and not in](https://docs.python.org/3/reference/expressions.html#in)) for: 
```py
for letter in snowman_word: 
    if user_guess == letter: 
        return True
    return False
```

You can use this syntax to perform the above logic over a multiple kinds of iterables/sequences in Python. Later in the curriculum we will learn that they don't all have the same performance (Big O). 


For the second half of your conditional, it essentially checks if the letter has been guessed correctly already or not. Keep in mind that the helper function `get_letter_from_user` performs this logic already. It is a redundancy to check it here. You could remove this piece of code. 
```suggestion
        if user_input in snowman_word:
```